robot/Cumulus/tests/browser/donations_payments/create_donation_and_payment.robot

```diff
@@ -25,20 +25,24 @@ Create Donation from a Contact
     Click Special Related List Button  Payments    Schedule Payments
     Wait For Locator    frame    Create one or more Payments for this Opportunity
     Choose Frame    Create one or more Payments for this Opportunity
-    Enter Payment Schedule    ${No_of_payments}    ${intervel}    ${frequency}
     ${loc}    Get NPSP Locator    id    inputX
     Input Text    ${loc}    8/15/2018
+    Enter Payment Schedule    ${No_of_payments}    ${intervel}    ${frequency}
+    ${xpath}    Get NPSP Locator    button    Calculate Payments
+    Execute JavaScript    window.document.evaluate('${xpath}', document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue.scrollIntoView(true)
+    #Page Scroll To Locator    button    Calculate Payments
     Click Button With Value    Calculate Payments
     ${value}     Verify Payment Split   100    ${No_of_payments}
     Should be equal as strings    ${value}    ${No_of_payments}
     Verify Date Split    8/15/2018    ${No_of_payments}    ${intervel}
+    ${xpath}    Get NPSP Locator    button    Create Payments
+    Execute JavaScript    window.document.evaluate('${xpath}', document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue.scrollIntoView(true)    
```

Comment:
So this is where automation is scrolling down to adjust for the addition of the Cancel button that was put into the footer?

Reply:
yes